solidity/contracts/OpenRenewalManager.sol

```diff
@@ -14,22 +14,15 @@ interface IENS {
     function owner(bytes32 node) external view returns (address);
 }
 
-contract ReverseClaimer {
-    // namehash('addr.reverse')
-    bytes32 constant ADDR_REVERSE_NODE = 0x91d1777781884d03a6757a803996e38de2a42967fb37eeaca72729271025a9e2;
-
-    constructor(IENS ens, address claimant) {
-        IReverseRegistrar reverseRegistrar = IReverseRegistrar(ens.owner(ADDR_REVERSE_NODE));
-        reverseRegistrar.claim(claimant);
-    }
-}
-
 address constant ENS_TOKEN_WRAPPER = 0xD4416b13d2b3a9aBae7AcD5D6C2BbDBE25686401;
 address constant ENS_REGISTRY_WITH_FALLBACK = 0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e;
+IENS constant ENS = IENS(ENS_REGISTRY_WITH_FALLBACK);
+bytes32 constant ADDR_REVERSE_NODE = 0x91d1777781884d03a6757a803996e38de2a42967fb37eeaca72729271025a9e2; // namehash('addr.reverse')
 
-contract OpenRenewalManager is ReverseClaimer {
-    constructor() ReverseClaimer(IENS(ENS_REGISTRY_WITH_FALLBACK), msg.sender) {
-
+contract OpenRenewalManager {
+    constructor() {
+        IReverseRegistrar reverseRegistrar = IReverseRegistrar(ENS.owner(ADDR_REVERSE_NODE));
+        reverseRegistrar.claim(msg.sender);
```

Comment:
I think this doesn't do anything useful since `msg.sender` is Deterministic Deployment Proxy.  I think we would need to set the `reverseRegistrar` claimant to either a contract (ideal, as then we could make the reverse immutable) or some EOA address.  Alternatively, we just don't let this have a reverse (not great, but maybe reverse isn't worth the complexity).